M2/Macaulay2/packages/Macaulay2Doc/operators/tensor.m2

```diff
@@ -34,3 +34,33 @@ Node
     Example
       {1, 2} ** {10, 20, 30}
 ///
+
+doc ///
+  Key
+    symbol ⊠
+  Headline
+    a binary operator, usually used for external product
+  Description
+    Text
+      To insert this character in Emacs, you may press @KBD "C-x 8 RET"@
+      or @KBD "M-x insert-char"@ and then enter "squared times" in the
+      minibuffer.
+
+      Alternatively, you may press @KBD "C-x RET C-\\"@ or
+      @KBD "M-x set-input-method"@ and then enter "TeX" in the minibuffer.
+      Afterwards, typing "\boxtimes" will input the ⊠ character.  You may then
+      toggle the input method using @KBD "C-\\"@  or
+      @KBD "M-x toggle-input-method"@.
```

Comment:
Since these paragraphs are mostly repeated, it may be worthwhile to have an automated section printed at the bottom of unicode operators that explains how to enter them. e.g. something like the "For the programmer" section underneath operators, which explains how to define methods on them.

Also, I still think every unicode operator should be tied to an ascii method name so it can be still used without unicode entry (e.g. `**` and `tensor` for $\otimes$).

Reply:
I like the idea of an automated section -- I'll push a proposal here in a bit.

As for the ascii method name... I'm not sure if we should force a particular name for each operator, or just leave it up to package authors.  Maybe one package  has ⊠ to be a shortcut for `externalTensorProduct` and another for `strongProduct` , etc.

But yeah, we should probably provide an ascii version for each method that's actually defined.  I'll go ahead and add `dotProduct(Vector, Vector)` as a synonym for `Vector · Vector`.